### PR TITLE
925 permalinks

### LINKED ISSUE #925: CMS page markup
- [x] Markup for the default layout for pages in a section with a sidebar navigation menu
- [ ] Implementation

### PR DESCRIPTION
**Pull request by jorgeatgu**
Closes #925

## :v: What does this PR do?
Now when we reload the datasets URL's we don't lose the data.

## :mag: How should this be manually tested?
Go to any dataset and reload the browser, check that it has the title, your queries, recent queries, and you can see the table with the data.

### CODE REVIEW

File: app/javascript/gobierto_data/webapp/components/sets/SQLEditor.vue [hunk 1]
```diff
@@ -101,12 +106,13 @@ export default {
       this.$root.$emit('storeQuery', this.recentQueries)
     },
     getSlug() {
-      this.endPointSlug = '/api/v1/data/datasets';
+      this.endPointSlug = `/api/v1/data/datasets/${this.$route.params.id}/meta`
```
Comment (Crashillo) [inline app/javascript/gobierto_data/webapp/components/sets/SQLEditor.vue]: baseUrls out

File: app/javascript/gobierto_data/webapp/pages/DataSets.vue [hunk 2]
```diff
@@ -25,29 +28,30 @@ export default {
...
     this.$root.$on('reloadQueries', this.getQueries)
     this.numberId = this.$route.params.numberId
 
+
     this.userId = getUserId()
   },
   methods: {
     getQueries() {
       this.urlPath = location.origin
       this.endPoint = '/api/v1/data/queries?filter[dataset_id]='
```
Comment (Crashillo) [inline app/javascript/gobierto_data/webapp/pages/DataSets.vue]: baseUrls out

File: app/javascript/gobierto_data/webapp/pages/DataSets.vue [hunk 3]
```diff
@@ -56,13 +60,21 @@ export default {
     },
     getData() {
       this.urlPath = location.origin
-      this.endPoint = '/api/v1/data/datasets/'
+      this.endPoint = `/api/v1/data/datasets/${this.$route.params.id}/meta`
```
Comment (Crashillo) [inline app/javascript/gobierto_data/webapp/pages/DataSets.vue]: baseUrls out

**Review by Crashillo — CHANGES_REQUESTED** [review]

File: app/javascript/gobierto_data/webapp/pages/DataSets.vue [hunk 4]
```diff
@@ -56,13 +62,22 @@ export default {
...
       this.url = `${this.urlPath}${this.endPoint}`
       axios
         .get(this.url)
         .then(response => {
           this.rawData = response.data
-          this.titleDataset = this.rawData.data[0].attributes.name
+          this.titleDataset = this.rawData.data.attributes.name
+          this.idDataset = this.rawData.data.id
```
Comment (ferblape) [inline app/javascript/gobierto_data/webapp/pages/DataSets.vue]: two attributes for the same thing? `idDataset` and `datasetId`

File: app/javascript/gobierto_data/webapp/pages/DataSets.vue [hunk 5]
```diff
@@ -56,13 +62,22 @@ export default {
...
+          this.titleDataset = this.rawData.data.attributes.name
```
Comment (ferblape) [inline app/javascript/gobierto_data/webapp/pages/DataSets.vue]: duplicated

**Review by entantoencuanto — APPROVED** [review]

**Pull request merged**